frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -708,6 +712,11 @@ pub mod pallet {
 
 		/// The weight of the pallet.
 		type WeightInfo: WeightInfo;
+
+		/// The minimum amount a solution challenger must have to execute
+		/// `challange_submission`
+		#[pallet::constant]
+		type MinimumSlashableAmount: Get<BalanceOf<Self>>;
```

Comment:
Similarly, I think the slashable deposit of the challenger should be dynamic and should be simply the same as the amount of the deposit of the solution that you are going after.